prometheus/prometheus.spec

```diff
@@ -13,7 +13,7 @@
 
 Name:           prometheus
 Version:        2.14.0
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
Same comment as 1.1 PR. If nothing changed, the version doesn't need to be bumped.

Reply:
I rebuild all RPMs when doing the releases